files/cloud-init/nfs/cloud-config

```diff
@@ -41,17 +41,17 @@ runcmd:
   #
   # Create Raid5 Array
   #
-  # NOTE: We need to sleep 5 minutes here to ensure the drives have been attached
-  #
-  # - sleep 300
```

Comment:
So this need just went away? Or does it continue to work given the UUID does not change? Just trying to understand why this was removed

Reply:
It has been in the code commented for long. This was removed as cleanup, I can add it back let me know.